### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by musm**
I ran into issues and was debugging, until I realized  `julia-buildpkg` was missing, presumably this also requires the step `setup-julia`

### CODE REVIEW

**Comment by SaschaMann** [comment]
Those aren't strictly needed to deploy docs. One can use the default Julia version that GHA ships, or deploy pre-built docs.

**Comment by musm** [comment]
Hmm, my docs kept failing because my package HDF5 wasn't precompiling because it was missing the `julia-buildpkg` stage.

File: README.md [hunk 1]
```diff
@@ -15,7 +15,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: julia-actions/julia-docdeploy@releases/v1
+      - uses: julia-actions/setup-julia@latest
```
Comment (SaschaMann) [inline README.md:18]:
```suggestion
```

**Comment by SaschaMann** [comment]
We can add the buildpkg step but remove setup-julia in that case

**Comment by SaschaMann** [comment]
Thanks!

**Comment by musm** [comment]
Thank you!

**Pull request merged**